gatsby-browser.js

```diff
@@ -8,8 +8,8 @@
 
 import React from "react"
 import { ThemeProvider } from "emotion-theming"
-import theme from "@rebass/preset"
 
+import theme from "./src/theme"
 import "./src/reset.css"
```

Comment:
May effect ba ang order ng imports ng css? Or reset.css muna before our custom theme?

Also, baka pwede na madelete ang comments above 😄

Reply:
moving reset css to global in emotion

Reply:
Maybe use a third party library for this? So we won't have to worry about manually updating reset.css. :grin:  https://www.npmjs.com/package/emotion-reset

Reply:
inspected the css reset used on the lib seems lacking IMHO. I was using this in the css file https://github.com/csstools/sanitize.css seems updated and modern.

Reply:
I just have to translate this to a global style in emotion or build an open source NPM package!!!

Reply:
I will just use https://www.npmjs.com/package/emotion-reset for now

Reply:
Hi if that is just a reset file and wont change overtime, just don't include it on runtime like import "emotion-reset.css" Just include it on the index.html file.

<link rel="stylesheet" href="/some-where/emotion-reset.css" />

Reply:
@psongpin nvm my comment